### PR TITLE
modified import order and spacing in general

### PR DESCRIPTION
**Pull request by eduardogamboa11**

### CODE REVIEW

File: bios/views.py [hunk 1]
```diff
@@ -1,44 +1,41 @@
+from datetime import datetime, date
```
Comment (mmartinezintersys) [inline bios/views.py]: I will leave @HeloelH or @MMSaavedraC to review this block of imports, I don't remember what Tony recommended to put the third party library imports and the django imports.

File: bios/views.py [hunk 2]
```diff
@@ -206,36 +213,36 @@ def get_cost_type(token, name):
...
+    headers = {"authorization": "Bearer " + token}
     response = requests.request("GET", url+"SELECT name, KimbleOne__Resource__r.Name, KimbleOne__DeliveryGroup__r.KimbleOne__Account__r.Name, KimbleOne__StartDate__c, KimbleOne__ForecastP1EndDate__c, KimbleOne__ForecastP2EndDate__c, KimbleOne__ForecastP3EndDate__c, KimbleOne__UtilisationPercentage__c FROM KimbleOne__ActivityAssignment__c WHERE KimbleOne__DeliveryGroup__c != NULL AND KimbleOne__Resource__r.Name = '"+bio.name+"'", headers = headers)
     assignments = json.loads(response.text)
-    assignment_end_date = None
 
     try:
-        decodedAssignments = assignments['records']
-
         kimbleSet = set()
         for assignment in assignments['records']:
```
Comment (mmartinezintersys) [inline bios/views.py]: Add a new line before the for block

File: consultantmarket/views.py [hunk 3]
```diff
@@ -3,14 +3,13 @@
...
-from django.shortcuts import render, get_object_or_404
-from django.http import HttpResponse, HttpResponseRedirect, JsonResponse
-from django.views.decorators.csrf import csrf_exempt
 from django.conf import settings
-from django.shortcuts import redirect
 from django.contrib import messages
-from django.views.decorators.http import require_POST
 from django.core.mail import EmailMultiAlternatives
+from django.http import HttpResponseRedirect
+from django.shortcuts import render, redirect
```
Comment (MMSaavedraC) [inline consultantmarket/views.py]: Fix ordering:

from django.shortcuts import redirect, render

File: roster/views.py [hunk 4]
```diff
@@ -1,21 +1,19 @@
-from django.http import HttpResponse, HttpResponseRedirect, JsonResponse, Http404
-from django.contrib import messages
 import datetime
-from django.http import JsonResponse
-from django.shortcuts import get_object_or_404, render
+
+from django.conf import settings
+from django.core.mail import send_mail
+from django.http import JsonResponse, Http404
```
Comment (MMSaavedraC) [inline roster/views.py]: fix order:

from django.http import Http404, JsonResponse

File: roster/views.py [hunk 5]
```diff
@@ -1,21 +1,19 @@
...
+from django.shortcuts import get_object_or_404, render, redirect
```
Comment (MMSaavedraC) [inline roster/views.py]: Fix order:

from django.shortcuts import get_object_or_404, redirect, render

File: roster/views.py [hunk 6]
```diff
@@ -1,21 +1,19 @@
...
 from django.views.decorators.csrf import csrf_exempt
 from django.views.decorators.http import require_POST
-from django.shortcuts import redirect
-from django.core.mail import send_mail
-from django.conf import settings
+from django.urls import reverse
+
+from account.forms import saveProject
+from bios.models import Owner, Account, Project, Bio, ProjectDefault
```
Comment (MMSaavedraC) [inline roster/views.py]: Fix order:

from bios.models import Account, Bio, Owner, Project, ProjectDefault

File: search/views.py [hunk 7]
```diff
@@ -1,45 +1,46 @@
+from datetime import datetime, date
+
 from django.core.paginator import Paginator
-from django.shortcuts import get_object_or_404, render
+from django.shortcuts import get_object_or_404, render, redirect
```
Comment (MMSaavedraC) [inline search/views.py]: fix order:

from django.shortcuts import get_object_or_404, redirect, render

**Comment by eduardogamboa11** [comment]
Is this ready for merge @MMSaavedraC @mmartinezintersys ?

**Pull request merged**